src/components/ResultContainer.tsx

```diff
@@ -115,6 +117,13 @@ function ResultContainer({ response }: { response: QueryResponse | null }) {
                   subject.num_matching_phenotypic_sessions,
                   subject.num_matching_imaging_sessions,
                   subject.image_modal?.join(', '),
+                  subject.completed_pipelines
+                    ? Object.entries(subject.completed_pipelines)
+                        .flatMap(([name, versions]) =>
+                          (versions as string[]).map((version: string) => `${name} ${version}`)
+                        )
+                        .join(', ')
+                    : {},
                 ].join('\t')
```

Comment:
I think this is a bit much complexity for an inline piece of code here. 
I get that you're constructing the pipeline+version string array here
but compared to all the other lines above it is much harder to read / understand.

Factor out into a function?